docker-compose.yml

```diff
@@ -90,8 +90,8 @@ services:
     environment:
       KAFKA_BROKER_ID: 2
       KAFKA_ZOOKEEPER_CONNECT: zookeeper1:2181,zookeeper2:2181,zookeeper3:2181
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9093
-      KAFKA_DEFAULT_REPLICATION_FACTOR: 2    
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka2:9093
+      KAFKA_LOG_DIRS: /var/log/kafka
```

Comment:
It's good practice to precisely set the path for logs because the default path is `/tmp/kaka-logs/` regarding documentation -> [see here](https://docs.confluent.io/platform/current/installation/configuration/broker-configs.html#log-dir)